Mainframe3270/x3270.py

```diff
@@ -209,21 +209,24 @@ def set_screenshot_folder(self, path: str) -> None:
             logger.warn('Screenshots will be saved in "%s"' % self.imgfolder)
 
     @keyword("Take Screenshot")
-    def take_screenshot(self, height: int = 410, width: int = 670) -> str:
+    def take_screenshot(self, height: int = 410, width: int = 670, fnamepfx: str = "screenshot") -> str:
```

Comment:
@Harm10 , please rename this to `filename_prefix` to be more consistent with the naming of the other parameters in this keyword.